src/layouts/index.tsx

```diff
@@ -56,14 +61,14 @@ width: 100%;
 /* There must be an entry for each of these in indexToSessionStorageKeys
     of progress-tracker.tsx */
 const LINKSPATH = [
-  {path: "/", label: "HOME"},
-  {path: "/prescreen-1a/", label: "BACKGROUND"},
-  {path: "/prescreen-1b/", label: "EARNINGS"},
-  {path: "/prescreen-1c/", label: "EMPLOYMENT STATUS"},
-  {path: "/screen-2/", label: "RESULTS"},
-  {path: "/screen-2a/", label: "BENEFIT FORMULA"},
-  // {path: "/screen-2b/", label: "OVERPAYMENT"},
-  // {path: "/screen-2c/", label: "TAKE ACTION"}
+  {path: "/", label: "Home"},
+  {path: "/prescreen-1a/", label: "Background"},
+  {path: "/prescreen-1b/", label: "Earnings"},
+  {path: "/prescreen-1c/", label: "Employment Status"},
+  {path: "/screen-2/", label: "Results"},
+  {path: "/screen-2a/", label: "Benefit Forumula"}
+  // {path: "/screen-2b/", label: "Overpayment"},
+  // {path: "/screen-2c/", label: "Take Action"}
```

Comment:
The design has these in uppercase, did you mean to change them? I don't think we should.

Reply:
I did mean to change, the idea being to avoid what seems like unnecessary reformatting. (Why go through the process of converting `"HOME"` to `"Home"`, when the label can simply be stored and referenced as `"Home"`? And so on.) So, code like: 
```
labelLeft = `Previous: ${
  LINKSPATH[index - 1].label[0] +
  LINKSPATH[index - 1].label.slice(1).toLowerCase()
}`;
```
becomes: ``` labelLeft = `Previous: ${linkPrev.label}`; ```.

I guess I don't follow - what design are you referencing? Do you mean the Figma mockups? Or perhaps a code convention that I'm unaware of? (For instance, "variable names for constants should be all caps, like `LINKSPATH`")

Reply:
<img width="363" alt="image" src="https://user-images.githubusercontent.com/283343/77955507-4ff3d300-729e-11ea-9422-5f2f5f0c105c.png">

became
<img width="289" alt="image" src="https://user-images.githubusercontent.com/283343/77955484-47030180-729e-11ea-9078-82d654586fe9.png">